app/models/signature.rb

```diff
@@ -39,6 +39,73 @@ def time_ago
     time_ago_in_words(created_at)
   end
 
+  def to_petition_csv_line
+    [name, email, city, state_symbol, full_country_name]
+  end
+
+  def state_symbol
+    state_hash[state]
+  end
+
+  def full_country_name
+    IsoCountryCodes.find(country_code).name
+  end
+
+
+  def state_hash
+    states = {
+      "Alabama" => "AL",
+      "Alaska" => "AK",
+      "Arizona" => "AZ",
+      "Arkansas" => "AR",
+      "California" => "CA",
+      "Colorado" => "CO",
+      "Connecticut" => "CT",
+      "Delaware" => "DE",
+      "Florida" => "FL",
+      "Georgia" => "GA",
+      "Hawaii" => "HI",
+      "Idaho" => "ID",
+      "Illinois" => "IL",
+      "Indiana" => "IN",
+      "Iowa" => "IA",
+      "Kansas" => "KS",
+      "Kentucky" => "KY",
+      "Louisiana" => "LA",
+      "Maine" => "ME",
+      "Maryland" => "MD",
+      "Massachusetts" => "MA",
+      "Michigan" => "MI",
+      "Minnesota" => "MN",
+      "Mississippi" => "MS",
+      "Missouri" => "MO",
+      "Montana" => "MT",
+      "Nebraska" => "NE",
+      "Nevada" => "NV",
+      "New Hampshire" => "NH",
+      "New Jersey" => "NJ",
+      "New Mexico" => "NM",
+      "New York" => "NY",
+      "North Carolina" => "NC",
+      "North Dakota" => "ND",
+      "Ohio" => "OH",
+      "Oklahoma" => "OK",
+      "Oregon" => "OR",
+      "Pennsylvania" => "PA",
+      "Rhode Island" => "RI",
+      "South Carolina" => "SC",
+      "South Dakota" => "SD",
+      "Tennessee" => "TN",
+      "Texas" => "TX",
+      "Utah" => "UT",
+      "Vermont" => "VT",
+      "Virginia" => "VA",
+      "Washington" => "WA",
+      "West Virginia" => "WV",
+      "Wisconsin" => "WI",
+      "Wyoming" => "WY"
+    }
+  end
```

Comment:
seems like this belongs in its own model, since it can be used generically and doesn't have to do with singatures specifically